src/corn-model.ts

```diff
@@ -69,6 +70,38 @@ export const addWormsSparse = () => {
   addWorms(8, 8, 40, 80, 20);
 };
 
+function addTrapCrop(rows: number, columns: number, rowStart: number, colStart: number, spacing: number) {
+  for (let row = 0; row < rows; row++) {
+    for (let column = 0; column < columns; column++) {
+      const seed = variedPlants.createAgent();
+      seed.setLocation({
+          x: rowStart + (column * spacing) + (row % 2 === 0 ? 6 : 0),
+          y: colStart + (row * spacing) + (column % 2 === 0 ? 4 : 0),
+      });
+      seed.set('infected', false);
+      env.addAgent(seed);
+    }
+  }
+}
+
+export const addTrapCropDense = () => {
+  addTrapCrop(12, 11, 25, 20, 38);
+};
+
+export const addTrapCropSparse = () => {
+  addTrapCrop(7, 7, 30, 50, 60);
+};
+
+export function infectInitialCorn(quantity: number) {
```

Comment:
I think this can come out - I removed it when working with worms since we have infestation, not infection, as the health metric.

Reply:
Good catch. There were merge conflicts when I rebased my branch on top of yours which I tried to resolve properly, but apparently this one snuck through. One of us should remove it in some upcoming PR that requires changes in this neighborhood.